docs/admin/upgrade/index.md

```diff
@@ -0,0 +1,11 @@
+# Upgrading
+
+Guidelines about upgrading CrateDB clusters.
+
+```{toctree}
+:maxdepth: 2
+
+Guidelines <planning>
+rolling
+full
```

Comment:
I've adjusted the patch correspondingly to use "Guidelines" as the label within the primary navigation. The page title itself hasn't been changed, and is still "General Upgrade Guidelines".

![image](https://github.com/crate/cratedb-guide/assets/453543/92270a3d-2922-4b45-b30d-7bb10bddea2d)

Reply:
One could just move the "Guidelines" into "Upgrading", as the Execution part just links to Rolling / Full Upgrade

Reply:
We have to admit we are only looking at the most prominent details mostly on the 1st level of this documentation tree now, with only occasional detours into the 2nd and 3rd levels. We will carry that over to a subsequent iteration, thanks.